signs-work-order-map/types/map.ts

```diff
@@ -0,0 +1,107 @@
+export interface LatLon {
+  latitude: number | undefined;
+  longitude: number | undefined;
+}
+
+export interface MapProps {
+  location: {
+    longitude: number | undefined;
+    latitude: number | undefined;
+  };
+  signs: Sign[];
+}
+
+export interface Sign {
+  id: string;
+  lng: number;
+  lat: number;
```

Comment:
Chia and I discussed an edge case error I ran into when I clicked on the **Add Location** button (even though feature is out of scope). I wonder if these lat/lon need to be optional?